apps/condo/domains/ticket/access/Ticket.js

```diff
@@ -123,7 +121,39 @@ async function canManageTickets ({ authentication: { item: user }, operation, it
             deletedAt: null,
         })
 
-        return residents.length > 0
+        if (residents.length === 0) {
+            return false
+        }
+
+        if (operation === 'create') {
+            return true
+        } else if (operation === 'update') {
+            let inaccessibleUpdatedFields
+
+            if (ticket.createdBy === user.id) {
+                inaccessibleUpdatedFields = omit(originalInput, ['dv', 'sender', 'details', 'reviewValue', 'reviewComment'])
+            } else {
+                const ticketContactId = get(ticket, 'contact')
+                if (!ticket.canReadByResident) return false
+                if (!ticketContactId) return false
+
+                const ticketContact = await getById('Contact', ticketContactId)
+                if (user.phone !== ticketContact.phone) {
```

Comment:
Tricky access :) do we really need it?)

Reply:
We need to give residents access to update `reviewComment` and `reviewValue` in order to review such tickets.